### PR TITLE
documentation update pretagging

### PR DESCRIPTION
**Pull request by IgorSusmelj**
### Changes

- Minor changes to active-learning documentation
- Minor updates to docstring of ApiWorkflowClient
- Added docker documentation for pretagging

### CODE REVIEW

File: docs/source/docker/advanced/pretagging.rst [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+    The plot shows the detected bounding boxes from the pretagging overlayed
+    on the image. Use the debug feature to figure out whether the pretagging 
+    mechanism works properly on your dataset.
+
+
+Pretagging for Selection
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can also use pretagging to guide the data selection process. This can beh
```
Comment (philippmwirth) [inline docs/source/docker/advanced/pretagging.rst:118]: Typo: beh

File: docs/source/docker/advanced/pretagging.rst [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+helpful if you for example only care about images where there is at least one
+person.
+
+To create such a pretagging selection mechanism you need to create a config file.
+
+For the example of selecting only images with >1 person we can create
```
Comment (philippmwirth) [inline docs/source/docker/advanced/pretagging.rst:124]: I need to double check the implementation but pretty sure it's >=

File: docs/source/getting_started/active_learning.rst [hunk 3]
```diff
@@ -95,22 +96,23 @@ Let's configure the sampling request and request an initial selection next:
...
-~~~~~~~~~~~~~~~~~~~~~~
+----------------------
 
 After you have annotated your initial selection of images, you can train a model
 on them. The trained model can then be used to figure out, with which images it 
 has problems. These images can then be added to the labeled dataset.
 
-To do active learning with Lightly, you will need the `ApiWorkflowClient` and `ActiveLearningAgent` from before.
+To continue with active learning with Lightly, you will need the `ApiWorkflowClient` and `ActiveLearningAgent` from before.
+If you perform the next selection step in a new file you have to initialize the client and agent again.
```
Comment (philippmwirth) [inline docs/source/getting_started/active_learning.rst:115]: Yes, that's better

File: lightly/api/api_workflow_client.py [hunk 4]
```diff
@@ -25,15 +25,23 @@
 
 
 class ApiWorkflowClient(_UploadEmbeddingsMixin, _SamplingMixin, _UploadDatasetMixin, _DatasetsMixin):
-    """Provides a uniform interface to communicate with the api and run workflows including multiple API calls
+    """Provides a uniform interface to communicate with the api 
+    
+    The APIWorkflowClient is used to communicaate with the Lightly API. The client
+    can run also more complex workflows which include multiple API calls at once.
+    
+    The Client can be used in combination withe the active-learning agent.
```
Comment (philippmwirth) [inline lightly/api/api_workflow_client.py:33]: Typo: withe

File: docs/source/docker/advanced/pretagging.rst (hunk 2 above)
Comment (IgorSusmelj) [reply docs/source/docker/advanced/pretagging.rst:124]: yes, you're right. It's >=1

File: docs/source/docker/advanced/pretagging.rst (hunk 1 above)
Comment (philippmwirth) [reply docs/source/docker/advanced/pretagging.rst:122]: @IgorSusmelj can you fix this, too?

File: docs/source/docker/advanced/pretagging.rst [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+.. code-block:: json
+
+    [
+        {
+            "filename": "0000000095.png",
+            "boxes": [
+                [
+                    869.5512084960938,
+                    153.78746032714844,
```
[the remark this answers is not in the document]
Comment (philippmwirth) [reply docs/source/docker/advanced/pretagging.rst:56]: @IgorSusmelj I checked and the boxes go from 0 to 1

**Review by philippmwirth — APPROVED** [review]

**Pull request merged**